configure.ac

```diff
@@ -70,12 +70,46 @@ OPAL_LOAD_PLATFORM
 # Start it up
 #
 
+OPAL_CONFIGURE_SETUP
+opal_show_title "Configuring project_name_long"
+opal_show_subtitle "Prerequisites"
+
 AC_CHECK_PROG([PERL],[perl],[perl],[no])
 AS_IF([test "X$PERL" = "Xno"],
       [AC_MSG_ERROR(["Open MPI requires perl. Aborting"])])
 
-OPAL_CONFIGURE_SETUP
-opal_show_title "Configuring project_name_long"
+dnl Intentionally not putting an AC HELP_STRING here so that this option
+dnl does not show up in "./configure --help" output.
+AC_ARG_ENABLE([dist])
+
+AC_PATH_PROG([PYTHON], [python])
+
+AC_PATH_PROG([PANDOC], [pandoc])
+
+happy=0
+AS_IF([test -n "$PANDOC"],
```

Comment:
Why do we need --enable-dist again?  --enable-dist is *EVIL*; it makes "make distcheck" not work properly.  Much better would be to just abort during "make dist" if the pandoc string is the "fake" pandoc.